File: programmers/js/level00/120871.js
// 저주의 숫자 3
function solution(n) {
  let count = 0;
  
  for (let i = 1; ; i++) {
    if (i % 3 !== 0 && !String(i).includes('3')) {
      count++;
    }
    
    if (count === n) {
      return i;
    }
  }
}

File: programmers/java/level00/120871.java
// 저주의 숫자 3

class Solution {
    public int solution(int n) {
        int count = 0;
        int result = 0;
        
        for (int i = 1; count < n; i++) {
            if (i % 3 != 0 && !String.valueOf(i).contains("3")) {
                count++;
                result = i;
            }
        }
        
        return result;
    }
}